From 2629d773681b53ed659a6009c15fcf01e7cd3e02 Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Wed, 11 Mar 2015 10:19:32 +0800
Subject: [PATCH] Board: at91sam9x5ek: remove allow recovery

As the at91sam9x5ek don't has pin definition for recovery, so
remove the allow recovery option.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
---
 board/at91sam9x5ek/Config.in.board | 2 --
 1 file changed, 2 deletions(-)

diff --git a/board/at91sam9x5ek/Config.in.board b/board/at91sam9x5ek/Config.in.board
index fed2042..d114241 100644
--- a/board/at91sam9x5ek/Config.in.board
+++ b/board/at91sam9x5ek/Config.in.board
@@ -9,8 +9,6 @@ config	CONFIG_AT91SAM9X5EK
 	select ALLOW_CPU_CLK_400MHZ
 	select ALLOW_CRYSTAL_12_000MHZ
 	select ALLOW_BOOT_FROM_DATAFLASH_CS0
-	select ALLOW_DATAFLASH_RECOVERY
-	select ALLOW_NANDFLASH_RECOVERY
 	select ALLOW_PIO3
 	select CPU_HAS_PMECC
 	select CONFIG_HAS_HW_INFO
-- 
1.9.1

